style.css

```diff
@@ -713,289 +665,42 @@ ul li {
-    top: 146px;
-    color: #344563;
-  }
-
-  .lang {
-    position: relative;
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    padding: 0;
-    gap: 8px;
-    width: 210px;
-    height: 24px;
-    left: 0;
-    top: 0;
-    margin-left: 0;
-    border-radius: 0;
-  }
-
-  .cardone button {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    padding: 12px;
-    gap: 10px;
-    position: relative;
-    width: 129px;
-    height: 48px;
-    margin-top: 0;
-    left: 0;
-    background: #fff;
-    border: 1px solid #6070ff;
-    border-radius: 8px;
-  }
-
-  .card2 {
-    position: absolute;
-    height: 496px;
-    width: 1156px;
-    left: 142px;
-    top: 780px;
-    background: #fff;
-    border: 1px solid #dfe1e6;
-    border-radius: 24px;
-  }
-
-  .card2image {
-    background-image: url(img/Snapshoot_Portfoliocard2.png);
-    background-repeat: no-repeat;
-    position: absolute;
-    width: 544px;
-    height: 448px;
-    left: 590px;
-    top: 24px;
-    border-radius: 8px;
-  }
-
-  .card2txt {
-    display: flex;
-    flex-direction: column;
-    align-items: flex-start;
-    padding: 24px;
-    gap: 24px;
-    position: absolute;
-    width: 544px;
-    height: 445px;
-    left: 24px;
-    top: 24px;
-  }
-
-  .card3 {
-    position: absolute;
-    height: 496px;
-    width: 1156px;
-    left: 142px;
-    top: 1418px;
-    background: #fff;
-    border: 1px solid #dfe1e6;
-    border-radius: 24px;
-  }
-
-  .card3image {
-    background-image: url(img/Snapshoot_Portfoliocard3.png);
-    background-repeat: no-repeat;
-    position: absolute;
-    width: 544px;
-    height: 448px;
-    left: 24px;
-    top: 24px;
-    border-radius: 8px;
-  }
-
-  .card3txt {
-    display: flex;
-    flex-direction: column;
-    align-items: flex-start;
-    padding: 24px;
-    gap: 24px;
-    position: absolute;
-    width: 544px;
-    height: 445px;
-    left: 590px;
-    top: 24px;
-  }
-
-  .card4 {
-    position: absolute;
-    height: 496px;
-    width: 1156px;
-    left: 142px;
-    top: 2056px;
-    background: #fff;
-    border: 1px solid #dfe1e6;
-    border-radius: 24px;
-  }
-
-  .card4image {
-    background-image: url(img/Snapshoot_Portfoliocard4.png);
-    background-repeat: no-repeat;
-    position: absolute;
-    width: 544px;
-    height: 448px;
-    left: 590px;
-    top: 24px;
-    border-radius: 8px;
-  }
-
-  .card4txt {
-    display: flex;
-    flex-direction: column;
-    align-items: flex-start;
-    padding: 24px;
-    gap: 24px;
-    position: absolute;
-    width: 544px;
-    height: 445px;
-    left: 24px;
-    top: 24px;
-  }
+  /* ---------about page---------------------- */
 
   .about {
-    position: absolute;
-    width: 1440px;
-    height: 924px;
-    left: 0;
-    top: 3610px;
-    background: #fff;
-    border-radius: 0 10% 0 0/0 10% 0 0;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 4.8rem;
+    padding: 2.8rem;
   }
```

Comment:
- [ ] The about myself takes 40% of the width while the language part takes 60% with margin between them. Your design does not look the exact way as we have it on the figma design. Kindly look into that.

### Your Design
![your-design-about](https://github.com/ntimanu/Portfolio/assets/26597432/36d7dbfc-f9b3-4a44-8709-940fecb95ae5)

### Figma Design
![temp-design-about](https://github.com/ntimanu/Portfolio/assets/26597432/08bc7d7e-d887-48c4-8647-55167396a111)


- [ ] The framework and skill underlines span across the whole 60% width, not only the width of the text. Kindly fix that too.

### Your Design
![your-underline](https://github.com/ntimanu/Portfolio/assets/26597432/403bfc6c-bda0-4dba-906e-c892aedcec65)

### Figma Design
![temp-underline](https://github.com/ntimanu/Portfolio/assets/26597432/ddb09380-21e3-4dec-b17c-31696daa92d4)

Reply:
Done